data/ecosystems/m/minotaur.toml

```diff
@@ -4,113 +4,18 @@ title = "Minotaur"
 sub_ecosystems = []
 
 github_organizations = [
-  "https://github.com/ErgoGravity",
+
   "https://github.com/minotaur-ergo",
-  "https://github.com/rosen-bridge",
+
 ]
 
 # Repositories
-[[repo]]
-url = "https://github.com/ErgoGravity/ergo-luport-executor"
-
-[[repo]]
-url = "https://github.com/ErgoGravity/ergo-susy-proxy"
-
-[[repo]]
-url = "https://github.com/ErgoGravity/gateway"
-
-[[repo]]
-url = "https://github.com/ErgoGravity/gateway-proxy"
-
-[[repo]]
-url = "https://github.com/ErgoGravity/gravity-core"
-
-[[repo]]
-url = "https://github.com/ErgoGravity/gravity-node-data-extractor"
-
-[[repo]]
-url = "https://github.com/ErgoGravity/startup-script"
```

Comment:
can you move these to a new eco?

Reply:
These were for a bridge that pre-dates Rosen, built by the same team. The bridge was abandoned due to the wormhole hack so this makes the most sense to include to show the development history of the team (as inevitably some of the logic/understanding gained from these were used in Rosen)